Explorer/Assets/DCL/AvatarRendering/Wearables/ApplicationParametersWearablesProvider.cs

```diff
@@ -65,14 +65,27 @@ await UniTask.WhenAll(RequestPointersAsync(pointers, BodyShape.MALE, ct),
                 }
             }
 
-            if (appArgs.TryGetValue(AppArgsFlags.SELF_PREVIEW_BUILDER_COLLECTION, out string? collectionId))
+            if (appArgs.TryGetValue(AppArgsFlags.SELF_PREVIEW_BUILDER_COLLECTIONS, out string? collectionsCsv))
             {
-                return await source.GetAsync(pageSize, pageNumber, ct, sortingField, orderBy, category, collectionType, name, results,
-                    loadingArguments: new CommonLoadingArguments(
-                        builderDTOsUrl.Replace(BUILDER_DTO_URL_COL_ID, collectionId),
-                        cancellationTokenSource: new CancellationTokenSource()
-                    ),
-                    needsBuilderAPISigning: true);
+                string[] collections = collectionsCsv!.Split(',', StringSplitOptions.RemoveEmptyEntries)
+                                                   .ToArray();
+
+                results ??= new List<IWearable>();
+                var localBuffer = new List<IWearable>();
```

Comment:
You can use a ListPool for this

Reply:
great advice, tackled at https://github.com/decentraland/unity-explorer/pull/3379/commits/4ffce7e0766bcfc6ae26dfec0a9705be55c66258